app/views/anthologies/_form.html.erb

```diff
@@ -13,7 +13,7 @@
         <div class="form-group">
           <%= f.label :banner %>
           <%= f.file_field :banner, class: 'form-control' %>
-        <i>The banner will be 3000 x 420 pixels </i>
+        <i>The banner should be a png, jpg, or jpeg and should be less than 1 MB</i>
```

Comment:
OK, and please add the new recommended dimensions.

Reply:
How about: "Your banner image should be a png, a gif or a jpg, 1100 pixels (or less) in width, and smaller than 1MB".